recipe/meta.yaml

```
@@ -43,13 +44,12 @@ about:
   license_family: MIT
   license_file: LICENSE.txt
   summary: Simple, fast, extensible JSON encoder/decoder for Python
-
   description: |
     simplejson is a simple, fast, complete, correct and extensible
     JSON <https://json.org> encoder and decoder for Python 2.5+ and
     Python 3.3+. It is pure Python code with no dependencies, but includes
     an optional C extension for a serious speed boost.
-  doc_url: https://simplejson.readthedocs.io/
+  doc_url: https://readthedocs.org/projects/simplejson/
```

```suggestion
  doc_url: https://simplejson.readthedocs.io/
```
I would keep the previous link.

---

Done thanks! The Linter God isn't always so kind 😂